WalletWasabi/Blockchain/Analysis/BlockchainAnalyzer.cs

```diff
@@ -9,7 +9,7 @@ namespace WalletWasabi.Blockchain.Analysis;
 
 public class BlockchainAnalyzer
 {
-	public static long[] StdDenoms = new[]
+	public static readonly long[] StdDenoms = new[]
```

Comment:
Please stop fixing unrelated things in PRs. This is making the review harder and messing up the history, also introducing inconsistency. You can create PRs that are fixing typos only or replace `static` with `readonly` are fine but it has to be done consistently everywhere in the entire code space.

Reply:
This can stay here but in future PRs.